mistletoe/latex_renderer.py

```diff
@@ -51,9 +56,9 @@ def render_escape_sequence(self, token):
         return self.render_inner(token)
 
     def render_raw_text(self, token, escape=True):
-        return (token.content.replace('$', '\$').replace('#', '\#')
-                             .replace('{', '\{').replace('}', '\}')
-                             .replace('&', '\&')) if escape else token.content
+        return (token.content.replace('$', '\\$').replace('#', '\\#')
```

Comment:
These double escaped back-slashes are not actually necessary here for Python, are they? I propose not to change these lines then.